posts/2021/08/low-code-workshop.md

```diff
@@ -0,0 +1,120 @@
+<!--
+.. title: Low-code contributions through Github
+.. slug: low-code-contributions-through-Github
+.. date: 2021-08-10 18:00:00 UTC-00:00
+.. author: Isabela Presedo-Floyd, Mars Lee, Melissa Weber Mendonça, Tony Fast
+.. tags: Jupyter, NumPy, Accessibility
+.. category:
+.. link:
+.. description:
+.. type: text
+-->
+
+
+# Low-code contributions through Github
+
+Healthy, inclusive communities are critical to impactful open source projects. 
+A challenge for established projects is that the history and implicit technical debt increase the barrier to contribute to significant portions of code base.
+The literacy of large code bases happens over time through incremental contributions, and we'll discuss a format that can begin this journey.
+
+At quansight labs, we are motivated to provide opportunities for new contributor's to experience open source community work regardless of their software literacy. 
```

Comment:
Q,L Uppercase for Quansight and Labs maybe ?

Reply:
```suggestion
At Quansight Labs, we are motivated to provide opportunities for new contributor's to experience open source community work regardless of their software literacy. 
```